### DISCUSSION TITLE
Some requests are failing due to PrismaClientValidationError: Unknown arg for virtual generated column

### DISCUSSION THREAD

**Discussion by amiregelz** [prisma/prisma#16826] [Q&A]
### Bug description

I am using Planetscale.
I ran a schema migration and created a virtual generated column.
My backend is Cloud Functions.

I get these errors for a lot of users, but it doesn't reproduce for me.
Is it possible that the client works for some connections but doesn't recognize the column for others?

I made sure I ran:
npx prisma db pull
npx prisma generate

### How to reproduce

Not sure

### Prisma information

```prisma
// Add your schema.prisma
```

```ts
// Add your code using Prisma Client
```

### Environment & setup

NodeJS Firebase Cloud Functions

### Prisma Version

3.15.0

**Comment by pantharshit00** [comment]
Hello @amiregelz,

Can you please try this again with the latest version of Prisma?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
